core/dbt/graph/selector.py

```diff
@@ -45,11 +47,14 @@ def __init__(self, node_spec):
             self.selector_value = node_spec
 
 
-class SELECTOR_FILTERS(object):
+class SELECTOR_FILTERS(str, Enum):
```

Comment:
why subclass `str` and `Enum` here?

Reply:
If you subclass `str` as well, this works:
```
In [1]: from enum import Enum

In [2]: class MyEnum(str, Enum):
   ...:     a = 'a'
   ...:     b = 'b'

In [3]: MyEnum.a == 'a'
Out[3]: True

In [4]: class OtherEnum(Enum):
   ...:     a = 'a'
   ...:     b = 'b'

In [5]: OtherEnum.a == 'a'
Out[5]: False
```

It's a small difference, but it just makes comparisons so much easier when your enums aren't opaque values. I was inspired by `enum.IntEnum` which is this, but for ints.